tests/Application/PathResolverTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Application;
+
+use NunoMaduro\PhpInsights\Application\PathResolver;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+
+final class PathResolverTest extends TestCase
+{
+    public function testResolvePathWithoutDirectorySeparator(): void
+    {
+        $path = 'app';
+        $targetPath = getcwd() . DIRECTORY_SEPARATOR . $path;
+
+        $input = new ArrayInput(
+            [
+                'paths' => [$path],
+            ],
+            new InputDefinition([
+                new InputArgument('paths'),
+            ])
+        );
+
+        $paths = PathResolver::resolve($input);
+
+        self::assertEquals($paths, [$targetPath]);
+    }
+
+    public function testResolvePathWithDirectorySeparator(): void
+    {
+        $path = '/app';
+        $sanitizedPath = getcwd() . $path;
```

Comment:
I think it's incorrect. 

If I provide an absolute path (like `/app`), this shouldn't be resolved as `current working dir + app`.